man/systemd.network.xml

```diff
@@ -796,6 +796,33 @@
       </variablelist>
   </refsect1>
 
+    <refsect1>
+    <title>[AddressLabel] Section Options</title>
```

Comment:
if this is an IPv6-only concept I think this should be prefixed with "IPv6", i.e. become `[IPv6AddressLabel]` or so in the config files.